src/components/scenes/SendScene2.tsx

```diff
@@ -1252,24 +1252,28 @@ const SendComponent = (props: Props): React.ReactElement => {
       } catch (e: unknown) {
         resetSlider()
         console.log(e)
-        if (e instanceof Error) {
-          let message = sprintf(lstrings.transaction_failure_message, e.message)
-          e.message = 'broadcastError'
-          if (e.name === 'ErrorAlgoRecipientNotActivated') {
+        const error = e instanceof Error ? e : new Error(String(e))
+        if (error != null) {
```

Comment:
We don't need the `if (error != null) {` check, because the line above guarantees that this is impossible.